jekyll/_cci2/vs-code-extension-overview.adoc

```diff
@@ -211,6 +213,16 @@ The following steps show how to test run a pipeline from the VS Code extension:
 . For each project, there is a button to trigger the config test run, and check boxes for each workflow and job for you to choose which parts of your config you want to run. Select your options, and click icon:play[]. Any local change you make to your `.circleci/config.yml` file will be included in the test run.
 . The pipeline is now visible in your link:https://app.circleci.com/pipelines/[dashboard] in the CircleCI web app, and the Pipelines panel in the VS Code extension.
 
+[#feature-settings]
+=== Feature Settings
+
+Access to this feature can be controlled **at the project level** or **at the org level**.
+
+* Project level settings can be found under menu:Project Settings[VS Code integration]. Toggle the "Run pipelines with unversioned config" option to **Off** to opt-out. 
+* Organization level settings can be found under menu:Organization Settings[Advanced]. Toggle "Trigger pipelines with unversioned config" option to **Off** to opt-out.
+  Organization level settings can be changed only by organization admins, and override project settings. 
```

Comment:
I'm not confident in any of this wording, if there is anything you'd like to change please go ahead!